packages/expo-cli/src/commands/eject/Eject.ts

```diff
@@ -180,18 +202,20 @@ async function ejectToBareAsync(projectRoot, options) {
   await fse.writeFile(path.resolve('app.json'), JSON.stringify(appJson, null, 2));
   log(chalk.green('Wrote to app.json, please update it manually in the future.'));
 
-  let defaultDependencies = {};
+  let defaultDependencies: StringObject = {};
 
   /**
    * Extract the template and copy it over
    */
   try {
-    let tempDir = temporary.directory();
+    const tempDir = temporary.directory();
     await Exp.extractTemplateAppAsync(templateSpec, tempDir, appJson);
     fse.copySync(path.join(tempDir, 'ios'), path.join(projectRoot, 'ios'));
     fse.copySync(path.join(tempDir, 'android'), path.join(projectRoot, 'android'));
-    let packageJson = fse.readJsonSync(path.join(tempDir, 'package.json'));
-    defaultDependencies = packageJson.dependencies;
+    const { dependencies } = JsonFile.read(path.join(tempDir, 'package.json'), { json5: true });
```

Comment:
```suggestion
    const { dependencies } = JsonFile.read(path.join(tempDir, 'package.json'));
```
`package.json` can not be JSON5 (neither npm nor Yarn accepts it).